index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+
+        <nav class="navbar">
+            
+                <a href="#">John Doe</a>
+                <a href="#"><img src="./Microverse-Student-Project-toolbar/Tertiary/Enabled.png" alt="menu icon"></a>
+            
+        </nav>
```

Comment:
- [ ] :wave: kindly look at the screenshot provided below
- [ ]  Your navbar isn't closing fully please fix this.

![Screenshot from 2022-06-28 17-28-06](https://user-images.githubusercontent.com/66230306/176548693-53e6a8c9-d2c0-40c2-b569-1a1fb3bd5c1c.png)